app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,37 @@
+<div class="signup-page">
+  <h2>Sign up</h2>
+
+  <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+    <%= render "devise/shared/error_messages", resource: resource %>
+
+    <div class="field">
+      <%= f.label :name %><br />
+      <%= f.text_field :name, autofocus: true, autocomplete: "name", class: "signup-form" %>
+    </div>
+
+    <div class="field">
+      <%= f.label :email %><br />
+      <%= f.email_field :email, autofocus: true, autocomplete: "email", class: "signup-form" %>
+    </div>
+
+    <div class="field">
+      <%= f.label :password %>
+      <% if @minimum_password_length %>
+      <em>(<%= @minimum_password_length %> characters minimum)</em>
+      <% end %><br />
+      <%= f.password_field :password, autocomplete: "new-password", class: "signup-form" %>
+    </div>
+
+    <div class="field">
+      <%= f.label :password_confirmation %><br />
+      <%= f.password_field :password_confirmation, autocomplete: "new-password", class: "signup-form" %>
```

Comment:
- [ ] The User model returns errors like invalid password length, and passwords not matching. For a better experience, instead of having the page do nothing, kindly ensure these errors are displayed on this form when a user tries to submit invalid details.

- [ ] **[OPTIONAL]** Since all these fields are mandatory, for a better user experience I would recommend you add the `required: true` HTML attribute so that users know that these fields are mandatory. For example:

  ```
  <%= f.email_field :email, autofocus: true, autocomplete: "email", class: "signup-form", required: true %>
  ```

Reply:
Fixed